Library/Homebrew/unversioned_cask_checker.rb

```diff
@@ -46,6 +51,11 @@ def single_app_cask?
       apps.count == 1
     end
 
+    sig { returns(T::Boolean) }
+    def single_installer_cask?
+      installers.count == 1
+    end
```

Comment:
This doesn't appear to be used?

Reply:
@Bo98 I was simply duplicating the logic that was previously in the class, it may not be necessary.

Reply:
@bevanjkay Please remove unless you know it's necessary.